knowledge-base/kb-windowsformhost-rendering.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Windows Form Host is Not Rendering If the Panel is Collapsible
+description: "How to render Windows Form host if the Panel is collapsible"
+type: how-to
+page_title: How to render Windows Form Host if the Panel is Collapsible in RadRibbonView for WPF
```

Comment:
This is a copy-paste from the "description" property. The "title", "page_title", and "description" properties should cover as much as possible any user input when doing a search both in our documentation, as well as via a browser search.